hack/tools/go.mod

```diff
@@ -1,16 +1,16 @@
 module k8s.io/test-infra/hack/tools
 
-go 1.18
+go 1.19
```

Comment:
would it make sense to just upgrade to 1.21?

Reply:
Prow uses go version 1.19. See the commit message in https://github.com/kubernetes/test-infra/pull/30418/commits/c3dc429c19e8265d7bba56a7ce664d840015bcea

Reply:
Looks like it was updated to 1.21 last week. Will update.